docs/source/concept_guides/deferring_execution.mdx

```diff
@@ -16,14 +16,43 @@ When you run your usual script, instructions are executed in order. Using 🤗 A
 GPUs at the same time introduces a complication: while each process executes all instructions in order, some may be
 faster than others.
 
+For this tutorial we will be utilizing both the [`Accelerator`] object and the [`PartialState`] object. The former is
+is the high-level API that you will use most of the time when using this library, the latter is the low-level API that
+you can use to defer the execution of some instructions to specific processes directly *without* the need to create an 
+[`Accelerator`] object first.
+
+To import and instantiate either do:
+
+```py
+from accelerate import Accelerator
+from accelerate.state import PartialState
+
+accelerator = Accelerator()
+partial_state = PartialState()
+```
+
+These two objects will be referenced as such throughout the rest of this tutorial.
+
+<Tip>
+
+    Every function that is shown from the `Accelerator` object in this tutorial can be found in the `PartialState` object as well.
+
+</Tip>
```

Comment:
Isn't it the other way round? Also no need for indentation.

Reply:
I suppose it would make more sense to show off the `PartialState` since we want users using that, will adjust!